jekyll-docs/datastores.md

```diff
@@ -36,7 +36,7 @@ ds1volume@datastore1
 We can check that volume is created in the datastore1
 
 ```
-$esx ls /vmfs/volumes/datastore1/dockvols
+$esx ls /vmfs/volumes/datastore1/ds1volume
```

Comment:
This isn't correct. The correct one is as follows:
/vmfs/volumes/datastore1/<tenant-uuid>/ds1volume
where tenant-uuid is default tenant uuid or user defined.

But I think here were need to put it in a simpler way. // CC  @shaominchen